src/PointwiseFunctions/ScalarTensor/ConstraintDampingTags.hpp

```diff
@@ -0,0 +1,77 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "Options/String.hpp"
+#include "PointwiseFunctions/ConstraintDamping/DampingFunction.hpp"
+
+/// \cond
+namespace ScalarTensor::OptionTags {
+struct Group;
+}  // namespace ScalarTensor::OptionTags
+/// \endcond
+
+namespace ScalarTensor {
+namespace OptionTags {
+template <size_t VolumeDim, typename Fr>
```

Comment:
Could you add a brief documentation string here? One option is to move tho docs from below in Tags to here, then use `\copydocs ScalarTensor::OptionTags::DampingFunctionGamma1` as the docs on the tag in the `Tags` namespace. (I always forget about this feature, which is why we haven't used it more, but it seems like a low-effort way to have more dox coverage :) )